VaticanBAV/et/BAVet131.xml

```diff
@@ -27,52 +27,1023 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus from="85r" to="85v"/>
+                                    <title type="complete" xml:lang="gez">Chant for Wednesday</title>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                <msItem xml:id="ms_i5.4">
+                                    <locus from="85v" to="86r"/>
+                                    <title type="complete" xml:lang="gez">Chant for Thursday</title>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                <msItem xml:id="ms_i5.5">
+                                    <locus from="86r" to="86v"/>
+                                    <title type="complete" xml:lang="gez">Chant for Friday</title>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                <msItem xml:id="ms_i5.6">
+                                    <locus from="86v" to="87r"/>
+                                    <title type="complete" xml:lang="gez">Chant for Saturday</title>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i6">
+                                <locus from="87r" to="89r"/>
+                                <title type="complete" xml:lang="gez" ref="LIT6489SchoolChants#Aryam">አርባዕት፡ ዘሰናብት።</title>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    ቃልየ፡ አጽምዕ፡ እግዚኦ፡ ሃሌ፡ <del rend="expunctuated">ሃሌ፡</del>ሉያ፡ ሃሌ፡ ሉያ፡ ወለቡ፡ ጽራኅየ፡ ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ 
+                                    ወአጽምአኒ፡ ቃለ፡ ስእለትየ፡ ንጉሥየኒ፡ ወአምላኪየኒ፨ ሃሌ፤ ሉያ፤
+                                </incipit>
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i7">
+                                <locus from="89r" to="91v"/>
+                                <title type="complete" xml:lang="gez">Troparia called አርያም፡ ዘቀዳሚ፡ ዜማ፡, for different feast days</title>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    ሃሌ፡ ሉያ፡ ለአብ፡ ሃሌ፡ ሉያ፡ ለወልድ፡ ሃሌ፡ ሉያ፡ <surplus>መወ</surplus>ለመንፈስ፡ ቅዱስ፡ ዮሐንስኒ፡ ሀሎ፡ ያጠምቅ፡ በኄኖን፡ በቅሩበ፡ ሳሌም፡ 
+                                    በማዕዶተ፡ ዮርዳኖስ፡ ወብርሃነ፡ ጽድቅሰ፡ ውእቱ፡ ኢየሱስ፡ ክርስቶስ፨
+                                </incipit>
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i8">
+                                <locus from="91v" to="97v"/>
+                                <title type="complete" xml:lang="gez" ref="LIT6289Semani">Chant for the office called ፫ት፡ ዘዘወትር፡</title>
```

Comment:
I am almost 100 % sure that this is simply LIT6489SchoolChants#Salast. Unfortunately, the incipit only covers the psalm and not the beginning of the antiphons, which would make it possible to identify the text with absolute certainty.